packages/shared/utils/testing.ts

```diff
@@ -0,0 +1,187 @@
+import type {
+  CallStack,
+  loadedRegions,
+  Location,
+  MappedLocation,
+  Message,
+  MessageLevel,
+  MessageSource,
+  newSource,
+  ObjectId,
+  PauseData,
+  PauseId,
+  PointDescription,
+  SourceId,
+  SourceKind,
+  TimeStampedPoint,
+  TimeStampedPointRange,
+  Value,
+} from "@replayio/protocol";
+
+export const DEFAULT_SOURCE_ID = "fake-source-id";
+export const DEFAULT_SOURCE_URL = "fake-source-url";
+
+let uidCounter = 0;
+
+export type MessageTuple = ["Console.newMessage", Message];
+
+export function createConsoleMessage({
```

Comment:
I moved these methods into the shared package so the demo app tests can use them too.